libs/priors/bbhnet/priors/priors.py

```diff
@@ -120,12 +119,33 @@ def nonspin_bbh() -> BBHPriorDict:
     return prior
 
 
+def spin_bbh() -> BBHPriorDict:
+    prior = uniform_extrinsic()
+    prior["mass_1"] = Uniform(5, 100, unit=msun)
+    prior["mass_2"] = Uniform(5, 100, unit=msun)
+    prior["mass_ratio"] = Constraint(0, 1)
+    prior["redshift"] = UniformSourceFrame(
+        0, 0.5, unit=mpc, name="redshift", cosmology=cosmo.Planck15
+    )
+    prior["psi"] = 0
+    prior["a_1"] = Uniform(0, 0.998)
+    prior["a_2"] = Uniform(0, 0.998)
+    prior["tilt_1"] = Sine(unit=rad)
+    prior["tilt_2"] = Sine(unit=rad)
+    prior["phi_12"] = Uniform(0, 2 * np.pi)
+    prior["phi_jl"] = 0
+
+    return prior
+
+
 def end_o3_ratesandpops() -> BBHPriorDict:
     prior = uniform_extrinsic()
     prior["mass_1"] = PowerLaw(alpha=-2.35, minimum=2, maximum=100, unit=msun)
     prior["mass_2"] = PowerLaw(alpha=1, minimum=2, maximum=100, unit=msun)
     prior["mass_ratio"] = Constraint(0.02, 1)
-    prior["redshift"] = UniformComovingVolume(0, 2, unit=mpc, name="redshift")
+    prior["redshift"] = UniformSourceFrame(
```

Comment:
Why did you change the prior here? It looks like the prior used in the rates and pops analysis was the comoving volume

Reply:
@deepchatterjeeligo has mentioned that Bilby's `UniformComovingVolume` is different than what is traditionally considered "UniformComovingVolume" (which seems weird). Can you link to where you've seen that prior implemented

Reply:
I'm basing this off what they describe [here](https://wiki.ligo.org/CBC/RatesPop/EndO3PopulationInjections), under the end o3 part of the BBH 'Broad' Injections. They say that p(z)~dV_c/dz * dt_source/dt * (1+z)^k. They chose k = 1, this it simplifies to p(z)~dV_c/dz, which is how bilby defines their uniform comoving volume prior. I'll try to find their implementation of it

Reply:
[This](https://git.ligo.org/RatesAndPopulations/lvc-rates-and-pop/-/blob/master/bin/endo3_distributions.py#L164) is where they implement their prior on z, but I'm having trouble finding documentation for that lal function.